demo/project/Build.scala

```diff
@@ -28,6 +28,8 @@ object DemoBuild extends Build {
     avroSchemaFiles := Seq(
       (resourceDirectory in Compile).value / "item.avsc"
     ),
+    scalaCustomTypes := Map("float"->"Double"),
```

Comment:
What would you think about defining these mapping relationships with a class rather than a string? That is, `scalaCustomTypes` would recieve a `Map[String, Class[_]]` like `Map("float" -> classOf[Double])` to preserve a bit more type safety.